packages/training/python/tests/test_post_training.py

```
@@ -0,0 +1,623 @@
+            mock_benchmark = MagicMock()
+            mock_benchmark.returncode = 0
+            mock_benchmark.stdout = "Benchmark complete"
+            
+            calls = []
+            def side_effect(cmd, **kwargs):
+                calls.append(cmd)
+                if isinstance(cmd, list) and cmd[0] == "which":
+                    return mock_which
+                return mock_benchmark
+            
+            with patch("subprocess.run", side_effect=side_effect) as mock_run:
```

Ruff (0.14.14)

[error] Local variable `mock_run` is assigned to but never used

Remove assignment to unused variable `mock_run`

(F841)